external/googleapis/protolists/iam_policy.list

```diff
@@ -1,4 +1,2 @@
-@com_google_googleapis//google/iam/v1/logging:audit_data.proto
-@com_google_googleapis//google/iam/v1:iam_policy.proto
-@com_google_googleapis//google/iam/v1:options.proto
-@com_google_googleapis//google/iam/v1:policy.proto
+@com_google_googleapis//google/iam/v2:deny.proto
```

Comment:
The name of the library and its contents do not really match anymore.  I think we got here through a series of reasonable decisions, but having a library called `iam_policy` which only contains the `v2` protos seems ... unfortunate.

Would `iam_v2` be a better name? or even `iam_v2_protos`?